domain/model/service/service.go

```diff
@@ -253,32 +253,32 @@ func (s *Service) CheckModelExists(ctx context.Context, modelUUID coremodel.UUID
 	return s.st.CheckModelExists(ctx, modelUUID)
 }
 
-// DefaultModelCloudNameAndCredential returns the default cloud name and
+// DefaultModelCloudInfoAndCredential returns the default cloud name, region name and
 // credential that should be used for newly created models that haven't had
 // either cloud or credential specified. If no default credential is available
 // the zero value of [credential.UUID] will be returned.
 //
 // The defaults that are sourced come from the controller's default model. If
 // there is a no controller model a [modelerrors.NotFound] error will be
 // returned.
-func (s *Service) DefaultModelCloudNameAndCredential(
+func (s *Service) DefaultModelCloudInfoAndCredential(
```

Comment:
This is only used from the model manager facade and because we don't use the credential part I think we should just drop it from the function signature?